jekyll/_cci2/deploy-to-npm-registry.adoc

```diff
@@ -32,9 +32,9 @@ jobs:
     publish:
     docker:
         - image: <docker-image-name-tag>
-        auth:
-            username: mydockerhub-user
-            password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+            auth:
```

Comment:
@kbatuigas those aren't extra indentations, those are pages that have tabs set instead of spaces for indentation and changing it on my settings in VS code for some reason didn't change the whole page. These whole pages need to be formatted differently for indentation.